src/main/menStore.java

```diff
@@ -0,0 +1,136 @@
+package main;
+
+import java.util.Scanner;
+
+public class menStore {
```

Comment:
please write class name starting with a capital letter

Reply:
ok, I'll fit it.